packages/trader/src/Modules/MT5/Containers/mt5-password-modal.jsx

```diff
@@ -40,7 +41,7 @@ const getIconFromType = type => {
     }
 };
 
-const MT5PasswordForm = ({ ...props }) => (
```

Comment:
how come this was changed?

Reply:
It was just an additional computation, didn't serve us any good if I'm not mistaken, I removed it.